static-build/components/TestPagination/index.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { h, FunctionalComponent } from 'preact';
+import {
+  $metaData,
+  $prevLinkData,
+  $nextLinkData,
+  $nav,
+  $svgLeft,
+  $svgRight,
+  $svgHolder,
+  $container,
+} from './styles.css';
+
+interface Props {
+  prev: PaginationData;
+  next: PaginationData;
+}
+
+const TestPagination: FunctionalComponent<Props> = ({ prev, next }: Props) => {
+  return (
+    <div class={$nav}>
+      <div>
+        {' '}
```

Comment:
```suggestion
```

cruft? not sure the purpose of this

Reply:
@argyleink I need to figure out where it came from